### PR TITLE
6840 ds sync send consent soln

### PR DESCRIPTION
**Pull request by siddharthsatyakam**

### CODE REVIEW

**Comment by nihr43** [comment]
please familiarize yourself with `git revert` and `git checkout`.  the first 7 commits here just add noise.

File: ds_status_sync.py [hunk 1]
```diff
@@ -308,7 +308,7 @@ def getstatus(self, stids: py.List[str]) -> py.List[object]:
         log.debug('getting status for %s:\ndata: %s\nheaders: %s',
                   stids, req.data, req.headers.items())
         s = self.__session
-        resp = s.send(s.prepare_request(req))  # type: ignore
```
Comment (nihr43) [inline ds_status_sync.py:311]: these were probably here for a reason.  please find out why and ensure its not broken

**Review by nihr43 — CHANGES_REQUESTED** [review]

File: ds_status_sync.py (hunk 1 above)
Comment (siddharthsatyakam) [reply ds_status_sync.py:311]: These are used to ignore the type of the assigned value to a variable of a particular type
https://github.com/python/mypy/issues/8823

As the assigned value here is not type cased before assignment and it was also confirmed by:
https://github.com/kumc-bmi/ds-determined-tools/runs/6217641639?check_suite_focus=true

So tried commenting them but it didn't work https://github.com/kumc-bmi/ds-determined-tools/commit/aa95aff990d7accf10d92c640d7b067470d447df

The github still complained of pylint error

So removed it, but can place it back or revert it if you say

**commit:** ds_status_sync.py: pylint error: error: Unused "type: ignore" comment… · kumc-bmi/ds-determined-tools@aa95aff
… at 311,343,423 lines

Comment (nihr43) [reply ds_status_sync.py:311]: i get that getting rid of them fixes pylint but does it break mypy?
does github run mypy as well?

**Comment by siddharthsatyakam** [comment]
> please familiarize yourself with `git revert` and `git checkout`. the first 7 commits here just add noise.

I will squash and merge at the end removing all unnessesary commit messages

File: ds_status_sync.py (hunk 1 above)
Comment (siddharthsatyakam) [reply ds_status_sync.py:311]: no presently github is not running mypy workflow.. Would you like me to implement those??

**Review by nihr43 — APPROVED** [review]
please squash and mind the merge commit message.  "6840 ds sync send consent soln" doesnt have much meaning to someone scanning the project history.

**Pull request merged**